src/Package/TestApp/Utility/ViewTreeDump.cs

```diff
@@ -36,27 +35,36 @@ internal class ViewTreeDump {
         }
 
         private static void CompareVisualTree(VisualTreeObject actual, VisualTreeObject expected, bool compareProperty = true) {
+            bool visible = true;
+            
             // compare
-            actual.Name.ShouldBeEquivalentTo(expected.Name);
+            actual.Name.Should().BeEquivalentTo(expected.Name);
```

Comment:
Actually, since this is a string, so `BeEquivalentTo` in this case is case-insensitive, leading/trailing whitespace ignoring comparison. Unless you need any of that, though, `Be` should be preferred.